contracts/GetWethValueInPoolBatchRequest.sol

```diff
@@ -143,10 +117,8 @@ contract GetWethValueInPoolBatchRequest {
     }
 
     ///Does not normalize to 18 decimals
-    function calculateV3VirtualReserves(
-        address pool
-    ) internal view returns (uint256 r_0, uint256 r_1) {
-        (uint160 sqrtPriceX96, , , , , , ) = IUniswapV3PoolState(pool).slot0();
+    function calculateV3VirtualReserves(address pool) internal view returns (uint256 r_0, uint256 r_1) {
```

Comment:
I don't believe this function is used anywhere, can we remove this?

Reply:
Yeah probably we can, lemme see and ill push if thats the case

Reply:
This has been done @0xOsiris